set_up_test_firebase.sh

```diff
@@ -0,0 +1,14 @@
+echo "import('./Tests/Fastfile')" | cat - fastlane/Fastfile | tee fastlane/Fastfile &> /dev/null
+
+readonly CONSTANT_TEAM_ID="<#teamId#>"
+readonly CONSTANT_STAGING_FIREBASE_APP_ID="<#stagingFirebaseAppId#>"
+readonly CONSTANT_FIREBASE_TESTER_GROUPS="<#group1#>, <#group2#>"
+readonly CONSTANT_MATCH_REPO="git@github.com:{organization}\/{repo}.git"
+
+readonly WORKING_DIR=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+MATCH_REPO_ESCAPED=$(echo "${MATCH_REPO//\//\\\/}")
+
+LC_ALL=C find $WORKING_DIR -type f -exec sed -i "" "s/$CONSTANT_TEAM_ID/4TWS7E2EPE/g" {} +
```

Comment:
Do we need to replace Team Id in the match flow? I remember that the test_testflight.sh does not require it 🤔 .
If we need maybe it's worth to inject from GitHub Secrets.

Reply:
The last time I tried, the workflow didn't work without `teamId` so I added it. I will update it to GitHub Secrets 👍

Reply:
@ducbm051291 Please fix this comment